src/pages/History/History.jsx

```diff
@@ -1,11 +1,29 @@
-import { Header, Sidebar } from '../../components/index';
+import { useEffect } from 'react';
+import { Header, Sidebar,VideoCard } from '../../components/index';
+import { useData } from '../../contexts/DataContext';
 import './History.css'
 export const History = () => {
+    const {history,getHistory}=useData();
+    useEffect(()=>{
+        getHistory();
+    },[])
```

Comment:
not really sure why there is getHistory() inside a useEffect u can directly map thru `history` which would have the history of all the videos user watched other than this all good

Reply:
`history` is an empty array and i am getting the previous watch history from server on the history page only and i have not done it in context